fetch.bs

```diff
@@ -4134,13 +4139,16 @@ the request.
  <a for=/>header value</a>) to <var>request</var>'s <a for=request>header list</a>.
 
  <li>
-  <p>If <var>request</var>'s <a for=request>priority</a> is null, then use <var>request</var>'s
-  <a for=request>initiator</a>, <a for=request>destination</a>, and
-  <a for=request>render-blocking</a> appropriately in setting <var>request</var>'s
-  <a for=request>priority</a> to a user-agent-defined object.
+  <p>If <var>request</var>'s <a for=request>internal priority</a> is null, then use
+  <var>request</var>'s <a for=request>priority</a>, <a for=request>initiator</a>,
+  <a for=request>destination</a>, and <a for=request>render-blocking</a> appropriately in setting
+  <var>request</var>'s <a for=request>internal priority</a> to an <a>implementation-defined</a>
+  object.
 
-  <p class=note>The user-agent-defined object could encompass stream weight and dependency for
-  HTTP/2, and equivalent information used to prioritize dispatch and processing of HTTP/1 fetches.
+  <p class=note>The <a for=request>internal priority</a> <a>implementation-defined</a> object
+  could encompass stream weight and dependency for HTTP/2, priorities used in
+  <a href=https://datatracker.ietf.org/doc/rfc9218/>HTTP extensible priorities</a>, and equivalent
+  information used to prioritize dispatch and processing of HTTP/1 fetches.
```

Comment:
We probably ought to say something about H/3.

Nit: I also think the original text here was correct in not repeating the field name.

Reply:
HTTP extensible priorities are used by H3 but it's not strictly limited to only H3.  Tweaked the language a bit to hopefully make it clearer.